sdks/python/apache_beam/runners/worker/log_handler.py

```diff
@@ -81,16 +83,36 @@ def connect(self):
         self._log_channel)
     return self._logging_stub.Logging(self._write_log_entries())
 
+  def map_log_level(self, level):
+    try:
+      return self.LOG_LEVEL_MAP[level]
+    except KeyError:
+      return max(
+          beam_level for python_level, beam_level in self.LOG_LEVEL_MAP.items()
+          if python_level <= level)
+
   def emit(self, record):
     log_entry = beam_fn_api_pb2.LogEntry()
-    log_entry.severity = self.LOG_LEVEL_MAP[record.levelno]
+    log_entry.severity = self.map_log_level(record.levelno)
     log_entry.message = self.format(record)
     log_entry.thread = record.threadName
     log_entry.log_location = record.module + '.' + record.funcName
```

Comment:
I'm not sure if it would be more natural to put filename:lineno instead of module.funcname but up to you. I understand that the LogEntry proto description lists Python to be module.funcname but that could be changed. Depends on what is more natural for Python developers.